Jenkinsfile

```diff
@@ -117,8 +104,10 @@ pipeline {
                                 withTools(params.TOOLS_VERSION) {
                                     withVenv {
                                         script {
-                                            withXTAG(["$VRD_TEST_RIG_TARGET"]) { adapterIDs ->
+                                            withXTAG(["$VRD_TEST_RIG_TARGET"]) { adapterIDs -> 
+                                                sh "tools/ci/config_hardware_str_50.sh " + adapterIDs[0]
                                                 sh "tools/ci/run_tests.sh " + adapterIDs[0]
+                                                sh "tools/ci/restore_factory_settings.sh " + adapterIDs[0]
```

Comment:
We can emulate the post/always of this by putting the restore in a try/finally block.

```suggestion
                                                try {
                                                    sh "tools/ci/config_hardware_str_50.sh " + adapterIDs[0]
                                                    sh "tools/ci/run_tests.sh " + adapterIDs[0]
                                                } finally {
                                                    sh "tools/ci/restore_factory_settings.sh " + adapterIDs[0]
                                                }
```

Reply:
Nice - thanks @BrennanGit